lib/repo/markus-git-shell.sh

```diff
@@ -63,23 +55,33 @@ source "${HOME}/.ssh/rc"
 [[ -z ${GIT_SHELL} ]] && write_log 'ERROR: GIT_SHELL not set' && exit 1
 [[ -z ${MARKUS_REPO_LOC_PATTERN} ]] && write_log 'ERROR: MARKUS_REPO_LOC_PATTERN not set' && exit 1
 
-REQUESTED_REPO_PATH=$(echo "${SSH_ORIGINAL_COMMAND}" | rev | cut -f1 -d' ' | rev) # ex: /csc108/group_1.git'
-REQUESTED_INSTANCE=$(basename "$(dirname "${REQUESTED_REPO_PATH}")") # ex: csc108
-REQUESTED_REPO_DIR=$(basename "${REQUESTED_REPO_PATH}") # ex: group_1.git'
-REQUESTED_REPO="${REQUESTED_REPO_DIR%.*}" # ex: group_1
+REQUESTED_REPO_PATH=$(echo "${SSH_ORIGINAL_COMMAND}" | rev | cut -f1 -d' ' | rev) # ex: /2021-02/csc108/group_1.git'
+_REQUESTED_DIR_PATH=$(dirname "${REQUESTED_REPO_PATH}") # ex: /2021/csc108
```

Comment:
I think the example should be `/2021-02/csc108` to be consistent with the previous examples?